services/identity/UsersService.ts

```diff
@@ -51,7 +61,7 @@ class UsersService {
   }
 
   async updateUserByGitHubId(githubId: string, user: string) {
```

Comment:
`user` should be of type `Partial<User>` in line 63. And then the first param in `this.repository.update()` in line 64 should be `user`

Reply:
It looks like it is fixed in https://github.com/isomerpages/isomercms-backend/pull/380/commits/9cfff3dfda32c06555203db4a289539581acf0c7 in PR #380